query/command/command.go

```diff
@@ -272,9 +276,8 @@ func (cmd *SelectCommand) Execute(context ExecutionContext) (CommandResult, erro
 		results <- result
 	}()
 	select {
-	case <-timeout:
-		return CommandResult{}, function.NewLimitError("Timeout while executing the query.",
-			context.Timeout, context.Timeout)
+	case <-ctx.Done():
+		return CommandResult{}, function.NewLimitError("Timeout while executing the query.", context.Timeout, context.Timeout)
```

Comment:
It's "actual" and "limit" which are the same here (since we don't go over by any meaningful amount, we don't record the actual time)